src/include/cell_inline.h

```diff
@@ -1449,7 +1449,8 @@ __wt_cell_unpack_delta_int(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *page_
     p += unpack_delta->key.__len;
 
     /* Optionally unpack the value if it exists. */
-    if (!F_ISSET(unpack_delta, WT_DELTA_INT_IS_DELETE)) {
+    if (!F_ISSET(unpack_delta, WT_CELL_ADDR_DEL)) {
```

Comment:
I don't understand this. The cell type is the in the value cell not in the unpack_delta